kale/embed/mogonet.py

```diff
@@ -0,0 +1,130 @@
+# =============================================================================
+# Author: Sina Tabakhi, sina.tabakhi@gmail.com
+# =============================================================================
+
+"""
+Construct a message passing network using PyTorch Geometric for the MOGONET method. MOGONET is a multiomics fusion
+framework for cancer classification and biomarker identification that utilizes supervised graph convolutional networks
+for omics datasets.
+
+This code is written by refactoring MOGONET code (https://github.com/txWang/MOGONET/blob/main/models.py) within
```

Comment:
the MOGONET code (check & fix all please)

Reply:
It has been corrected.